Commencement.Jobs.UpdateStudents/settings.job

```diff
@@ -0,0 +1 @@
+﻿{ "schedule": "0 5 * * *" }

```

Comment:
@jSylvestre I *think* you'll want to move the 5 over one space. so maybe "0 0 5 * *"